src/manage/ManageModel.js

```diff
@@ -0,0 +1,81 @@
+export default class QuestionModel {
+  constructor(db) {
+    this.db = db;
+    this.currentCategory = 'all';
+    this.newCategory = 'JavaScript';
+    this.newTitle = '';
+    this.newAnswer = '';
+    this.questionList = [];
+  }
+
+  get displayQuestionList() {
+    if (this.currentCategory === 'all') {
```

Comment:
전반적으로 이런 상수값을 문자열 string 그대로 사용하기 보다는 아래와 같이 사용하는게 유지보수에 좋아요. 다른곳에서도 사용한다면 별도 파일로 분리하면 됩니다.

```suggestion
const CATEGORY_TYPE = {
  ALL: 'all',
  // ... ETC
}
if (this.currentCategory === CATEGORY_TYPE.ALL) {
```